do

```diff
@@ -74,6 +75,10 @@ elif [ "$action" = "oc-push" ]; then
 elif [ "$action" = "oc-deploy" ]; then
   oc-deploy ${2-} ${3-} ${4-} ${5-}
 
+# Services
+elif [ "$action" = "data-service" ]; then
```

Comment:
`bash do data-service` will run the data service in a container locally.  It does require a `.env` file with correct credentials